meeting_minutes/workshop_01_03_2023.md

```diff
@@ -49,3 +49,9 @@ Project-Origin met with two implementers from Energy Origin (Github: [Energinet
   - Functionality to delegate access to third parties such as accountant to see what is going on in event stream and verify that against the value on ledger. 
 - How can we make the project progress more robust? Load balancing? training more people?
 
+### Decisions agreed
+- @wisbech & @guidmaster finds an estimate of required ressources
+- @soped2 invites to a workshop for the actual implementation between the production team and project-origin developers
+
+###
+- @mortenhouborg suggested that we should assign names to the above actions to keep momentum
```

Comment:
Names on the above actions has already been addressed in an e-mail sent by me 08/03 to all participants. At the time I assessed that doing so sufficiently informed the involved parties, as this summary rather serves to inform the community as a whole of progress on Project-Origin from the workshop. 

However, we have not agreed on responsibles for all of the above actions. For further work here I suggest to create issues for the tasks/actions ahead and assign people to those instead of processing these meeting minutes further,